src/index.css

```diff
@@ -16,12 +22,79 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+.container {
+  width: 70%;
+  position: relative;
+  left: 18%;
+  margin-top: 20px;
+}
+
+.result {
+  text-align: right;
+  padding: 16px 4px 16px 16px;
+  font-size: 18px;
+  font-weight: 500;
+  background-color: #858694;
+  color: #e0e0e0;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  padding: 10px;
+  margin-top: -10px;
+}
+
+.item1,
+.item2,
+.item3,
+.item5,
+.item6,
+.item7,
+.item9,
+.item10,
+.item11,
+.item13,
+.item14,
+.item15,
+.item17,
+.item18 {
+  background-color: #e0e0e0;
+  border: 1px solid #d3d3d3;
+}
```

Comment:
- [ ] Please give all calculator cells more height to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/176284035-51ccf525-187d-43bc-be1f-e114c754f0ae.png)  | ![image](https://user-images.githubusercontent.com/68796232/176283980-10d5165b-4cbc-4421-aa48-5c93acd72d4d.png)  |